docs/src/references/agent/software-management.md

```diff
@@ -21,13 +21,13 @@ The `software_list` operation API follows the [generic thin-edge rules for opera
 ### Operation registration
 
 The registration message of the `software_list` operation on a device:
-- must provide a `supportedTypes` list of the types of software package that can be installed on this device (e.g. `["apt", "docker"]`)
+- must provide a `types` list of the types of software package that can be installed on this device (e.g. `["apt", "docker"]`)
 - can provide a description of the operation and of each supported package type.
 
 ```sh te2mqtt formats="v1"
 tedge mqtt pub --retain 'te/device/child001///cmd/software_list' '{
     "description": "List software packages installed on the device",
-    "supportedTypes": [
+    "types": [
       "apt",
       "docker"
     ]
```

Comment:
Note, that currently these values are ignored.

Reply:
Are you sure? Because I found this doc change when updating the tedge-container-demo to use the new MQTT api and ran into this issue. I changed the child device to use the `types` property instead of `supportedTypes` and the expected types were sent back to Cumulocity IoT as expected.

Reply:
I added a note about this future feature in bae425ccbf28f25d7749b05be339f3836a08e763